packages/databyss-editor/lib/clipboardUtils/adjustSelectionToIncludeInlineAtomics.ts

```diff
@@ -25,15 +26,25 @@ export default ({
   }).filter(
     (r) =>
       r.marks.length &&
-      r.marks.filter((i) => Array.isArray(i) && i[0] === 'inlineTopic').length
+      r.marks.filter(
```

Comment:
you need the `.length` here, because `filter` still returns truthy if no results are found, eg:
`!![].filter(k => false) // === true`